src/apis/tasks.js

```diff
@@ -0,0 +1,113 @@
+import { getToken } from "../utils/utils";
+
+async function getTasks(apiEndPoint) {
+  const response = await fetch(
+    `${process.env.REACT_APP_SERVER_DOMAIN}/api/v1/tasks${apiEndPoint}`,
```

Comment:
define variable for base url and use variable instead of using env value directly.